__tests__/index.test.js

```diff
@@ -335,4 +335,99 @@ describe(`DELETE /api/comments/:comment_id tests`, () => {
             })
         })
     })
+})
+
+describe(`GET /api/articles tests V2 including queries  `, () => {
+    describe(`GET tests for sortby`, () => {
+        test(`/api/articles?sort_by=topic&order_by=asc returns correctly`, () => {
+            return request(app)
+            .get("/api/articles?sort_by=topic")
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body;
+                expect(articles)
+                .toBeSortedBy('topic', {descending: true})
+                articles.forEach(article => 
+                    expect(article).toEqual(
+                        expect.objectContaining({
+                            author: expect.any(String),
+                            title: expect.any(String),
+                            article_id: expect.any(Number),
+                            topic: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            comment_count: expect.any(String)
+                        })
+                    )
+                )
+                expect(articles.length).toBe(12)
+            })
+        })
+        test(`/api/articles?sort_by=topic&order_by=asc returns correctly`, () => {
+            return request(app)
+            .get("/api/articles?sort_by=topic&order_by=asc")
```

Comment:
Ideally you would checxk the other columns can be sort_by as well. 

It would be better to find the bug here, than when you're trying to get it to sort on the front end, and cant work out why only some of them are working.

Reply:
I am considering using template literals and a forEach statement for column headings in my test would that suffice?

Reply:
But im not implementing it,m i could have done it but I'm not